### DISCUSSION TITLE
Does CM6 not support vue3.2 (syntax sugar) +ts+vite ？

### DISCUSSION THREAD

**Discussion by ispaomoya** [surmon-china/vue-codemirror#161] [General]
in components
import VueCodemirror from 'vue-codemirror'
error
runtime-core.esm-bundler.js:38 [Vue warn]: Component is missing template or render function.

**Comment by surmon-china** [comment]
Hi! @ispaomoya,

I think this is a bug caused by the component registration name, which should have been fixed in the [v6.0.2](https://github.com/surmon-china/vue-codemirror/commit/7434c848a05c6259fdccc7b28eb1a51cc5c5344e) release.

**Comment by surmon-china** [comment]
If you are using it in a local component, please use ES module introductions instead of default:

```ts
import { Codemirror } from 'vue-codemirror'
```
```html
<Codemirror />
```

If you are using it globally, please import default:

```ts
import { createApp } from 'vue'
import VueCodemirror from 'vue-codemirror'

const app = createApp(App)
app.use(VueCodemirror)
```
